(DOCS-2753) Remove 'very' and 'highly' from docs (#12577)
DataDog/documentation: content/en/integrations/faq/collect-custom-windows-performance-counters-over-wmi.md

kind: faq
---

> Datadog's WMI integration is a versatile approach to collecting relevant metrics from your Windows environments. As long as you're able to query a WMI object for a numerical value, you'll be able to configure your dd-agent's WMI check to run the same query and collect its results as a metric (find a great guide [on how to retrieve WMI metrics][1]).

Reviewer: ```suggestion
Datadog's WMI integration is a versatile approach to collecting relevant metrics from your Windows environments. As long as you're able to query a WMI object for a numerical value, you can configure your dd-agent's WMI check to run the same query and collect its results as a metric. See the [how to retrieve WMI metrics][1] guide.
```
